db/migrate/20190121210000_create_recipes.rb

```diff
@@ -0,0 +1,18 @@
+# Create Recipes Table
+
+class CreateRecipes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :recipes do |t|
+      ## 
+      t.string :title,       null: false, default: ""
+      t.string :snipped,     null: false, default: ""
```

Comment:
I think this should be `snippet` or `description`. I also think it's reasonable to let it be null and leave off the default. 

I also think removing the default on the title line would be good, too. We don't want that to be null, but we don't want blank titles, either.